public/index.php

```diff
@@ -1,4 +1,7 @@
 <?php
+
+namespace GanttDashboard;
+
 use Phalcon\Di\FactoryDefault;
 
 error_reporting(E_ALL);
```

Comment:
change the error reporting based on environment
On local is fine to be E_ALL... but not on other environments